cypress/e2e/Survey/index.spec.ts

```diff
@@ -0,0 +1,34 @@
+const surveyScreenTestIds = {
+  backButton: 'survey__back-button',
+};
+
+describe('Survey screen', () => {
+  context('given the Back button is clicked', () => {
+    beforeEach(() => {
```

Comment:
As you just have 1 test case, do we need this beforeEach hook? I think we can put all of them together.

Reply:
@hoangmirs Thanks, I updated it in [9c4d78c](https://github.com/manh-t/react-survey/pull/71/commits/9c4d78c25ef33000c6a8fbfbef584da760b8baca)